unit_testing/cli/test_cli_sct_propseg.py

```diff
@@ -12,3 +14,10 @@ def test_sct_propseg_backwards_compat(script_runner):
     logger.debug(f"{ret.stderr}")
     assert ret.success
     assert ret.stderr == ''
+
+
+def test_sct_propseg_o_flag():
```

Comment:
Same comments re: subprocess and `tmp_path`.

Reply:
About subprocess: [Here](https://github.com/neuropoly/spinalcordtoolbox/blob/dc97d42730d7e603f83e54c68f242ad2bef7189b/spinalcordtoolbox/scripts/sct_propseg.py#L657-L676) sct_propseg needs the output of the parser as it is not called inside the 'main' function.

Reply:
Interesting... thanks for pointing that out! Yes, this is definitely inconsistent between scripts.